core/src/test/java/com/yandex/yoctodb/FieldValueExtractionTest.java

```diff
@@ -68,6 +76,42 @@ public void extractSortableFieldValue() throws IOException {
         assertEquals(from(0).toByteBuffer(), db.getFieldValue(0, FIELD_NAME));
     }
 
+    @Test
+    public void extractStoredFieldValue() throws IOException {
+        final DocumentProvider db = build(IndexOption.STORED);
+        assertEquals(from(0).toByteBuffer(), db.getFieldValue(0, FIELD_NAME));
+    }
+
+    @Test
+    public void extractStoredLong() throws IOException {
+        final DocumentProvider db = build(IndexOption.STORED);
+        final long storedValue = from(25L).toByteBuffer().getLong() ^ Long.MIN_VALUE;
+        assertEquals(storedValue, db.getLongValue(0, LONG_FIELD_NAME));
+    }
+
+    @Test
+    public void extractStoredInt() throws IOException {
+        final DocumentProvider db = build(IndexOption.STORED);
+        final long storedValue = from(25).toByteBuffer().getInt() ^ Integer.MIN_VALUE;
+        assertEquals(storedValue, db.getIntValue(0, INT_FIELD_NAME));
```

Comment:
Why not test equality directly to 25 associated to constant?

Reply:
Fixed